liiweb/views/legislation.py

```diff
@@ -26,8 +26,8 @@ def get_form(self):
             self.form_defaults = {"sort": "-date"}
         return super().get_form()
 
-    def get_base_queryset(self, *args, **kwargs):
-        qs = super().get_base_queryset(*args, **kwargs)
+    def get_queryset(self):
+        qs = super().get_queryset()
```

Comment:
I don't think this works, because facet data will be calculated on the non-variant queryset, which isn't correct.

`add_children` is a bit of a weird edge case because it actually should bypass the main queryset completely. I think it's reasonable for `add_children` to explicit load its own queryset since it has explicit requirements about it; it basically almost does it already.

Reply:
I've changed the queryset to use the unfiltered Legislation queryset.